resources/views/pages/graphs.blade.php

```diff
@@ -228,6 +228,24 @@ function drawChart() {
 
 	var chart = new google.visualization.LineChart(document.getElementById('ave_num_trainings_chart'));
 	chart.draw(data, options);
+
+	// Participants' preference for camp type (based on reviews)
+	var data = google.visualization.arrayToDataTable(<?php echo json_encode($camp_prefs); ?>);
+
+	var options = {
+		  fontSize: 14,
+		  hAxis: {
+			  minValue: 0,
+			  gridlines: {
+				  count: -1
+			  }
+		  },
+          legend: { position: 'none' }
+        };
+
+	var chart = new google.visualization.BarChart(document.getElementById('camp_preference_chart'));
+	chart.draw(data, options);
```

Comment:
With pr #103 you can simplify this to a function call to `drawGoogleChart`

Reply:
Yeah saw that. Will be sure to implement that.

Reply:
@blackshadev drawback to current implementation of `drawGoogleChart` is that you have to specify the column names and types in the view, instead of in the controller (where I think it fits better because it's part of the data logic). What's your opinion of this?

Reply:
Yeah I'd think it is better suited at the controller part. To me the reason for it being in JS is: 
 1. The library requires it to be there
 2. It is part of the displaying / view. 

But you require the data (columns) to be there, which is the responsibility of the controller. So it's better to do it there.
Maybe we can write a wrapper around it such that the wrapper injects the JS parts into the view, with the correct columns etc.

Reply:
Concluded that I can't be bothered to rewrite at the moment. Refactored it to call `drawGoogleChart`, we can further overhaul the graph system with the next update to it.